Description: mindframe: beside the lifted library in the qmd extension, with the extension's test helpers (#96)
Forwarded: https://github.com/breferrari/mindframe/issues/19
Last-Update: 2026-10-13

diff --git a/core/scripts/extensions/qmd/qmd-ignore.test.ts b/core/scripts/extensions/qmd/qmd-ignore.test.ts
--- a/core/scripts/extensions/qmd/qmd-ignore.test.ts
+++ b/core/scripts/extensions/qmd/qmd-ignore.test.ts
@@ -20,9 +20,9 @@
 	upsertIgnoreInYaml,
 	writeQmdIgnore,
 	qmdConfigPath,
-} from "../lib/qmd-ignore.ts";
+} from "./lib/qmd-ignore.ts";
 
-import { rmTemp } from "./_helpers.ts";
+import { rmTemp } from "./test-fakes.ts";
 
 const YAML_BASE = `collections:
   obsidian-mind:
